packages/remix-deno/server.ts

```diff
@@ -0,0 +1,69 @@
+import { createRequestHandler } from "@remix-run/server-runtime";
+import type { ServerBuild } from "@remix-run/server-runtime";
+
+import mime from "mime";
+
+export function defaultCacheControl(url: URL) {
+  if (url.pathname.startsWith("/build/")) {
+    return "public, max-age=31536000, immutable";
+  } else {
+    return "public, max-age=600";
+  }
+}
+
+export function createRequestHandlerWithStaticFiles({
+  build,
+  mode,
+  staticFiles: { cacheControl, publicDir } = {
+    cacheControl: defaultCacheControl,
+    publicDir: "./public"
+  }
+}: {
+  build: ServerBuild;
+  mode?: string;
+  staticFiles?: {
+    cacheControl?: string | ((url: URL) => string);
+    publicDir?: string;
+  };
+}) {
+  let remixHandler = createRequestHandler(build, {}, mode);
```

Comment:
Let's split this up into 3 separate functions:

- `createRequestHandler`
- `createRequestHandlerWithStaticFiles`
- `serveStaticFiles`